src/pages/getting-started/helpdesk-apps/index.mdx

```diff
@@ -0,0 +1,158 @@
+
+Connect your HelpDesk to an external tool or service, extend HelpDesk user interface, or automate various tasks. Configure different functionalities, integrate with the LiveChat ecosystem, or create your own custom actions for agents in HelpDesk.
+
+# App capabilities
+
+LiveChat provides developers with multiple tools which allow your app to do many different things, for example:
+
+- Integrate with other products and services
+- Extend HelpDesk user interfaces
+- Create your own custom actions for agents in HelpDesk
+- Automate tasks with webhooks
+- Let users sign in to your service with Sign in with LiveChat
+
+If you plan to make your app public for everyone, check out our [monetization](/monetization/how-to-monetize-apps) and distribution tools.
+
+# Creating HelpDesk apps
+
+## Step-by-step guide
+
+If you want to create a simple HelpDesk app in the Developer Console, follow the instructions we’ve prepared. We'll use an app named Your Buddy as our example.
+
+### Step 1 Getting started
+
+To start, you need to check if you are signed in to the <Placeholder id="DEVELOPER_CONSOLE_URL" />. You can <a href="https://accounts.livechat.com/signup" target="_blank">sign up here</a>. However, your LiveChat or HelpDesk credentials should log you in automatically. 
+
+The first step in the Developer Console is to go to the **Apps** tab and click the **Create new app +** button. Give your app a name and click the **Continue** button. The application's name can be changed later.
+
+<img
+  alt="Creating HelpDesk apps"
+  src="/images/getting-started/helpdesk-apps/livechat-helpdesk-apps-creating new app.png"
+  width="1459px"
+  height="771px"
+/>
+
+Next, you need to decide which product you want to build your app for. In this tutorial guide, we will take care of the HelpDesk product. However, if you would like to check out how to build LiveChat apps, find more details [here](/getting-started/livechat-apps). The logo of the chosen product will be visible on the left side of your screen, under the **Building Blocks** tab.
+
+<div style={{maxWidth: '450px'}}>
+  <img src="/images/getting-started/helpdesk-apps/livechat-helpdesk-apps-choosing-product.png" alt="Creating HelpDesk apps choosing the product" width="557px" height="386px"/>
+</div>
+
+If you do not have a HelpDesk account, you will be asked to activate it. If you want to activate HelpDesk and start a free trial, click **Activate HelpDesk** button.
+
+<div style={{maxWidth: '450px'}}>
+  <img src="/images/getting-started/helpdesk-apps/livechat-helpdesk-apps-activating account.png" alt="Creating HelpDesk apps activating account" width="279px" height="230px"/>
+</div>
+
+💡 **Need a helping hand?** Right after signing in to the Developer Console, you can launch the **App wizard**. We go there with guidelines for the LiveChat app creation process, but it can also be very helpful while exploring HelpDesk.
```

Comment:
```suggestion
💡 **Need a helping hand?** Right after signing in to the Developer Console, you can launch the **App Wizard**. We go there with guidelines for the LiveChat app creation process, but it can also be very helpful while exploring HelpDesk.
```